.circleci/config.yml

```diff
@@ -99,6 +99,8 @@ jobs:
       - run:
           name: Run Unit Tests
           command: bundle exec fastlane test_without_building name:UnitTests try_count:3 device:'<< parameters.device >>' ios-version:'<< parameters.ios-version >>'
+      - store_test_results:
+          path: build/results
       - ios/save-xcodebuild-artifacts:
           result-bundle-path: build/results
```

Comment:
⚠️  Seems that those 2 steps are kinda "conflicting", as the second one already includes the first one, as you can see [in its definition in the CircleCI orb here](https://github.com/wordpress-mobile/circleci-orbs/blob/trunk/src/ios/orb.yml#L355-L365).

Which not only leads to an unnecessary step, but also to the second one not finding any report once the first passed:
![image](https://user-images.githubusercontent.com/216089/121702104-de362c80-cad1-11eb-8ada-60b17c03856b.png)

Reply:
😢  Also sadly, CircleCI doesn't seem to show multiline errors properly either in its "Tests" tab:
![image](https://user-images.githubusercontent.com/216089/121702529-41c05a00-cad2-11eb-811b-dffa0f757a12.png)

Not sure if it's worth spending too much time here since we plan to migrate away from CircleCI soon-ish so that multiline issue might be solved there… or not

@jkmassel any idea how BuildKite might show this kind of test result in their UI? I don't think they have a dedicated "Test Results" tab like CircleCI does, right, so we will either have to log the XML files in stdout with the risk of being truncated, or just upload the test result XMLs as artefacts and let people download them to inspect failures?

---

In any case, it would be nice if we could find a way to print those multiline error messages from the XML to stdout, so that (1) they are visible in CircleCI log output directly with nice formatting (unlike Test tab which ignores newlines apparently) (2) we'll be ready to also be able to see them once we migrate to BuildKite (or another service that doesn't have a Tests tab like CCI). Sadly `trainer` can only generate JUnit XML and no other format like HTML or plain text (maybe with a simple-enough XSLT stylesheet and calling `xsltproc` we could get a nice output? 🤔 🤔 )

I'm ok to keep that for later and only keep the Tests tab without multiline support for now (I'm guessing this was a HACK week project and week is now over anyway… 😛 )

Reply:
Hey @AliSoftware I added that `store_test_results` because the `ios/save-xcodebuild-artifacts` step looks for reports in the `build/reports` folder wich does not exists. You can see that any other PR [does nothing](https://app.circleci.com/pipelines/github/woocommerce/woocommerce-ios/12376/workflows/11293f6f-a769-47b4-b58d-5701526555c6/jobs/22604) in that step. 

Do you think I should remove the `- ios/save-xcodebuild-artifacts:` step?

Reply:
Ah good point I missed the difference between `build/results` and `build/reports` there 😅 

We should keep that `save-xcodebuild-artefacts` step as it also upload crash reports that might have been encountered during test runs and other stuff so I think it makes more sense to keep it. So either we keep the solution you used here (and one of the 2 steps will not find anything), or you can check if it's easy to change the location for the trainer reports from `build/results` to `build/reports` if trainer provides that option. If not, another alternative could be to add a `mv build/results build/reports` step to do that 😛

Reply:
I think what we can do is to make `multi_scan` [to output](https://github.com/woocommerce/woocommerce-ios/blob/develop/fastlane/Fastfile#L814) in `build/reports` would that be ok?

Reply:
wfm